frontend/src/metabase/writeback/containers/ActionParametersInputForm/ActionParametersInputForm.tsx

```diff
@@ -51,11 +51,8 @@ function ActionParametersInputForm({
       actions.setSubmitting(true);
       const paramsWithDefaultValues = setDefaultValues(params, fieldSettings);
 
-      const formattedParams = formatParametersBeforeSubmit(
-        paramsWithDefaultValues,
-        formParams,
-      );
```

Comment:
the form component gives us the values in the right format already, so this whole function is unnecessary